documentation/source/Usecases/percona_galera_cluster.rst

```diff
@@ -0,0 +1,251 @@
+
+Percona Galera Cluster
+========================
+ 
+Running Percona Galera Cluster with OpenEBS
+--------------------------------------------
+
+This section provides detailed instructions on how to perform the following tasks.
+* Run a 3-node Percona Galera cluster with OpenEBS storage in a Kubernetes environment
+* Test data replication across the Percona Xtradb mysql instances.
+
+Galera Cluster
+----------------
+Percona XtraDB Cluster is an active/active highly available and scalable open source solution for MySQL clustering. It integrates Percona server and Percona XtraBackup with the Codership Galera library of MySQL high availability solutions in a single package. This folder contains the Kubernetes deployment specification YAMLs to setup the Galera cluster which includes the following.
+
+* A cluster service YAML which can be used for client connections (pxc-cluster)
+* The node deployment and service specification YAMLs to setup a 3-node replication cluster (pxc-node)
+
+The image used in these pods is *capttofu/percona_xtradb_cluster_5_6:beta*. When deployment is created, following activities occur in the given order.
+
+* The Percona Xtradb containers start
+* Runs an entrypoint script which does the following.
+
+  * Installs the MySQL system tables
+  * Sets up users
+  * Builds a list of servers that is used with the galera parameter *wsrep_cluster_address*. This is a list of running nodes that Galera uses for electing a node to obtain Single State Transfer (SST).
+
+Prerequisite
+-------------
+A fully configured multi-node Kubernetes cluster configured with the OpenEBS operator and OpenEBS storage classes. For instructions on applying the OpenEBS operator and recommended system configuration, see prerequisites, step 1 and step 2 in  the `MongoDB section`_.
+  
+  .. _MongoDB section: http://openebs.readthedocs.io/en/latest/Usecases/mongodb.html 
```

Comment:
Maybe this should point to a generic section and not to mongoDB.. just to avoid swtiching between two database docs..

Reply:
Done!